src/styles/index.css

```diff
@@ -6,4 +6,4 @@
 */
 @import '~@patternfly/patternfly/patternfly.css';
 @import '~@patternfly/patternfly/patternfly-addons.css';
-@import '~openshift-assisted-ui-lib/dist/index.css';
+@import '~openshift-assisted-ui-lib/dist/index-ocm.css';
```

Comment:
Is this still relevant? I don't see it being produced by ui-lib

Reply:
used to be but not anymore. We keep using `index.css`